packages/jade-compiler/lib/transpilers.js

```diff
@@ -42,6 +42,69 @@ var throwError = function (message, node) {
   throw new Error(message);
 };
 
+// Recursively search for anonymous helper functions and
+// record them
+// Arguments:
+// value: the tree created by the transpiler.
+// found: a map from helper function names to
+//    their function body.
+function searchNewArgs(value, found, foundEvents) {
```

Comment:
I'm not sure where I could make use of it, as the code I wrote doesn't create extra HTML.